activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/test/java/org/activiti/cloud/services/query/events/handlers/ProcessCompletedEventHandlerTest.java

```diff
@@ -85,11 +92,45 @@ public void handleShouldThrowExceptionWhenRelatedProcessInstanceIsNotFound() {
     }
 
     @Test
-    public void getHandledEventShouldReturnProcessCompletedEvent() {
+    void getHandledEventShouldReturnProcessCompletedEvent() {
         //when
         String handledEvent = handler.getHandledEvent();
 
         //then
         assertThat(handledEvent).isEqualTo(ProcessRuntimeEvent.ProcessEvents.PROCESS_COMPLETED.name());
     }
+
+    @Test
+    void handleShouldUpdateAssignedAndCreatedChildTasksAsCancelled() {
+        //given
+        ProcessInstanceImpl processInstance = new ProcessInstanceImpl();
+        processInstance.setId(UUID.randomUUID().toString());
+        CloudProcessCompletedEvent event = new CloudProcessCompletedEventImpl(processInstance);
+
+        ProcessInstanceEntity currentProcessInstanceEntity = mock(ProcessInstanceEntity.class);
+        TaskEntity createdTask = getTaskMock(TaskStatus.CREATED);
+        TaskEntity assignedTask = getTaskMock(TaskStatus.ASSIGNED);
+        TaskEntity suspendedTask = getTaskMock(TaskStatus.SUSPENDED);
+        when(currentProcessInstanceEntity.getTasks()).thenReturn(Set.of(createdTask, assignedTask, suspendedTask));
+        given(entityManager.find(ProcessInstanceEntity.class, event.getEntity().getId())).willReturn(
+            currentProcessInstanceEntity);
+
+        //when
+        handler.handle(event);
+
+        //then
```

Comment:
Can we also verify this https://github.com/Activiti/activiti-cloud/pull/1000/files#diff-7e8aa852cb6c0a1fd452f49e79e5f3d2c9bac969c7db14925d6261d9a88ec37eR67?

something like:
```
verify(entityManager).persist(createdTask);
verify(entityManager).persist(assignedTask);
verify(entityManager, never()).persist(suspendedTask);
```

Reply:
Good idea, thanks